### ISSUE TITLE
Provide documentation on how to structure the branch information

### ISSUE THREAD

**Issue by zFernand0** [octorelease/run-script#8]
Note: This might be a user-error issue. 😋 

I believed this snippet of code should have worked based on how the context is passed to the `getDependencies` function (but it doesn't - [failing build 😢 ](https://github.com/zFernand0/zowe-cli-scs-plugin/actions/runs/3176264986/jobs/5175360565))
https://github.com/zFernand0/zowe-cli-scs-plugin/pull/1/commits/1fc6d62a148401536e8aa20df29177194060d565
```js
        {
            name: "update-workflows",
            level: "patch",
            prerelease: true,
            devDependencies: ["@zowe/imperative"],
            channel: "zowe-v1-lts"
        }
```
https://github.com/zowe-actions/octorelease-script/blob/04ceb76b9da6193614b0a172117649016a17fca9/scripts/npmUpdate.ts#L39

---

Also, I believed this worked as well, but didn't - [failing build 😭 ](https://github.com/zFernand0/zowe-cli-scs-plugin/actions/runs/3176226865/jobs/5175275551)
https://github.com/zFernand0/zowe-cli-scs-plugin/pull/1/commits/ff8d77b70b90b1ed93877b2ca77ae278f54622dd

```js
        {
            name: "update-workflows",
            level: "patch",
            prerelease: true,
            devDependencies: {
                "@zowe/imperative": "zowe-v1-lts"
            }
        }
```

---

UPDATE: Seems like the second snippet works as expected ?
Build: https://github.com/zFernand0/zowe-cli-scs-plugin/pull/2/commits/f2fbdc6ee86ed1e44c906fcc4afd7de1302c121e
https://github.com/zFernand0/zowe-cli-scs-plugin/actions/runs/3176376079/jobs/5175599785

UPDATE 2: On a PR, it only uses the `base.ref` as the default branch to load context from
https://github.com/zowe-actions/octorelease-script/blob/04ceb76b9da6193614b0a172117649016a17fca9/src/index.ts#L32

**Comment by zFernand0** [comment]
TL;DR: User error
Closing! 😋
